### PR TITLE
New package: bird-1.5.0

### PR DESCRIPTION
**Pull request by hanspolo**

### CODE REVIEW

File: srcpkgs/bird/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+license="GPL-3"
+homepage="http://bird.network.cz"
+distfiles="ftp://bird.network.cz/pub/$pkgname/$pkgname-$version.tar.gz"
+hostmakedepends="flex"
+makedepends="ncurses-devel readline-devel"
+checksum=79d5c4254e25f5890626830ca42aae03735716bdf152ddd8368473d2b3c17b5d
+
+post_install() {
+	rm -rf ${DESTDIR}/usr/local
+}
```
Comment (Gottox) [inline srcpkgs/bird/template]: make my OCD happy and add a newline at the end ;)

File: srcpkgs/bird/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'bird'
+
+pkgname=bird
+version=1.5.0
+revision=1
+build_style=configure
```
Comment (Gottox) [inline srcpkgs/bird/template]: Use `gnu-configure` instead.

**Comment by Gottox** [comment]
The package is empty: https://travis-ci.org/voidlinux/void-packages/jobs/60368476#L791

**Comment by Gottox** [comment]
https://travis-ci.org/voidlinux/void-packages/jobs/60372286#L810

/etc/bird.conf should be a conf_file. Add this line:

```
conf_files="/etc/bird.conf"
```

**Comment by chneukirchen** [comment]
IPv6 support?
a runit service?

**Comment by Gottox** [comment]
I don't recommend subpackages here. Split the package into two independant packages bird and bird6.

**Comment by hanspolo** [comment]
I close this pr because of my fuck up.

**Pull request closed without merging**